crates/env/src/api.rs

```diff
@@ -183,8 +183,22 @@ where
     })
 }
 
-/// Writes the value to the contract storage under the given key and returns
-/// the size of pre-existing value at the specified key if any.
+/// The elder version of the function which is equivalent to the new one
+/// but returns nothing.
+///
+/// # Panics
+///
+/// - If the encode length of value exceeds the configured maximum value length of a storage entry.
```

Comment:
```suggestion
/// Writes the value to the contract storage under the given key.
///
/// # Panics
///
/// - If the encode length of value exceeds the configured maximum value length of a storage entry.
///
/// # Developer Note
///
/// This is equivalent to the new [`set_contract_storage`] method, but in order to maintain old
/// behavior it returns nothing.
```

You'll probably have to fix the method link so it points to the right thing